platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/crypto/CryptoStatic.java

```diff
@@ -44,27 +42,10 @@
 import java.math.BigInteger;
 import java.nio.file.Files;
 import java.nio.file.Path;
-import java.security.InvalidKeyException;
-import java.security.KeyPair;
-import java.security.KeyStore;
-import java.security.KeyStoreException;
-import java.security.NoSuchAlgorithmException;
-import java.security.NoSuchProviderException;
-import java.security.PublicKey;
-import java.security.SecureRandom;
-import java.security.Security;
-import java.security.Signature;
-import java.security.SignatureException;
-import java.security.UnrecoverableKeyException;
+import java.security.*;
```

Comment:
Our coding style guide is an extension of Google's and we do not allow imports to have wildcards (3.3.1) : https://google.github.io/styleguide/javaguide.html#s3.3-import-statements

I think there is an IDE setting which can be made to not use wildcards.

Reply:
Fixed